projects/control-service/projects/helm_charts/pipelines-control-service/templates/alertmanager.yaml

```diff
@@ -0,0 +1,25 @@
+{{- /*
+  Copyright 2021-2023 VMware, Inc.
+  SPDX-License-Identifier: Apache-2.0
+ */}}
+
+{{- if .Values.alertmanager.enabled }}
+apiVersion: monitoring.coreos.com/v1alpha1
+kind: AlertmanagerConfig
+metadata:
+  labels: {{- include "pipelines-control-service.labels" . | nindent 4 }}
+    {{- range $key, $value := .Values.alertmanager.metadata.labels }}
+    {{ $key }}: {{ $value | quote }}
+  {{- end }}
+  name: {{ .Values.alertmanager.metadata.name }}
+  namespace: {{ .Values.alertmanager.metadata.namespace }}
+spec:
+  receivers:
+    - emailConfigs: {{ toYaml (index .Values.alertmanager.spec.receivers 0).emailConfigs | nindent 8 }}
```

Comment:
why do you support an array for recievers but then only use the first one?

Reply:
https://docs.openshift.com/container-platform/4.10/rest_api/monitoring_apis/alertmanagerconfig-monitoring-coreos-com-v1alpha1.html#spec-receivers

The CRD definition is strict and will complain when installing the chart.